i18n/en.yaml

```diff
@@ -0,0 +1,15 @@
+# Learn how to use Date format (date, created, updated)
+# -> https://gohugo.io/functions/dateformat/
+
+posts:
+  title:
+    other: "Posts"
+  date:
+    other: "Jan, 2 2006"
+
+
+post:
+  created:
+    other: "Jan, 2 2006"
+  updated:
+    other: "Updated Jan, 2 2006"
```

Comment:
Can you convert this to use TOML instead of YAML? And move the comma after the day of the month instead of the month (ie `Jan 2, 2006`).

Reply:
Done